blazor/pager/customization.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Customization in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about how to customize the elements of Syncfusion Blazor Pager component and much more.
```

Comment:
description: Checkout here and learn about how to customize the elements of Syncfusion Blazor Pager component and much more.